CommandLineParams1.java

```diff
@@ -0,0 +1,7 @@
+public class CommandLineParams1 {
+	public static void main(String[] args){
+		for(int i = 0; i < args.length; i++){
+			System.out.println("�������� " + i + " = " + args[args.length-1-i]);
```

Comment:
It looks like compile shows me error about ACII code.
Maybe, would it be better to use ACII?